<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocalUserWidget</class>
 <widget class="QWidget" name="LocalUserWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QToolButton" name="sourceToolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="renameLabel">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Rename to:</string>
       </property>
       <property name="buddy">
        <cstring>renameLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLineEdit" name="descriptionLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="passwordLabel">
       <property name="text">
        <string>Password:</string>
       </property>
       <property name="buddy">
        <cstring>passwordLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="fullNameLabel">
       <property name="text">
        <string>Full name:</string>
       </property>
       <property name="buddy">
        <cstring>fullNameLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="fullNameLineEdit"/>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QLineEdit" name="confirmPasswordLineEdit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="renameLineEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="confirmPasswordLabel">
       <property name="text">
        <string>Confirm password:</string>
       </property>
       <property name="buddy">
        <cstring>confirmPasswordLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Description:</string>
       </property>
       <property name="buddy">
        <cstring>descriptionLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QLineEdit" name="passwordLineEdit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="userNameLabel">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>User Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="userComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Administrator (built-in)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Guest (built-in)</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="accountHorizontalLayout">
     <item>
      <widget class="QLabel" name="accountExpiresLabel">
       <property name="text">
        <string>Account expires:</string>
       </property>
       <property name="buddy">
        <cstring>accountExpiresDateEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="accountExpiresDateEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QSplitter" name="actionSplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="actionLabel">
      <property name="text">
       <string>Action:</string>
      </property>
      <property name="buddy">
       <cstring>actionComboBox</cstring>
      </property>
     </widget>
     <widget class="QComboBox" name="actionComboBox">
      <property name="currentText">
       <string>Create</string>
      </property>
      <item>
       <property name="text">
        <string>Create</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Replace</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Update</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Delete</string>
       </property>
      </item>
     </widget>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QVBoxLayout" name="userSettingsVerticalLayout">
     <item>
      <widget class="QCheckBox" name="changeLogonCheckBox">
       <property name="text">
        <string>User must change password at last logon</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="userCannotCheckBox">
       <property name="text">
        <string>User cannot change password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="passwordNeverCheckBox">
       <property name="text">
        <string>Password never expires</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="accountDisabledCheckBox">
       <property name="text">
        <string>Account is disabled</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="accountNeverCheckBox">
       <property name="text">
        <string>Account never expires</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>111</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>actionComboBox</tabstop>
  <tabstop>sourceToolButton</tabstop>
  <tabstop>renameLineEdit</tabstop>
  <tabstop>fullNameLineEdit</tabstop>
  <tabstop>descriptionLineEdit</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>confirmPasswordLineEdit</tabstop>
  <tabstop>changeLogonCheckBox</tabstop>
  <tabstop>userCannotCheckBox</tabstop>
  <tabstop>passwordNeverCheckBox</tabstop>
  <tabstop>accountDisabledCheckBox</tabstop>
  <tabstop>accountNeverCheckBox</tabstop>
  <tabstop>accountExpiresDateEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>accountNeverCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>accountExpiresDateEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>410</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
